src/qt/overviewpage.h

```diff
@@ -53,7 +53,6 @@ public Q_SLOTS:
 Q_SIGNALS:
     void transactionClicked(const QModelIndex &index);
     void enabledTorChanged();
-    void elysiumTransactionClicked(const uint256& txid);
```

Comment:
Please see description. Omni have removed this signal in latest versions of code. Do you intend to implement this signal?

Reply:
The event supposed to be emitted by clicked tx event on the overview page but it seems like the implementation isn't completed yet. after checking the Omnicore's code they intend to emit this in this function(https://github.com/OmniLayer/omnicore/blob/v0.8.1/src/qt/overviewpage.cpp#L362)(broken for now).

As I understand this provides the behavior that we click Omni tx on overview page and this will move to the Omni tx table(not sure if have other mechanic which do this instead) instead of the normal tx table. IMO, it improves the  UX then we should finish it instead of removing it completely. But the PR is for fixing the error messages and this is not an important feature then I think we should just keep it and wrap two connect() calls below then we can back to finish it later.

Reply:
Ok sure, I can wrap the elysium signals in the configure flag and leave them in for now. It should be implemented when possible though.

Reply:
Thanks